src/Components/Sliders/Popular/SliderPopular.js

```diff
@@ -3,48 +3,71 @@ import {NavLink} from 'react-router-dom'
 import PropTypes from 'prop-types'
 import './SliderPopular.css'
 import ProductCard from '../../ProductCard/ProductCard';
+import Slider from "react-slick"
 
-const SliderPopular = (props) => {
-    const productList = props.productsList.map(product => 
+class SliderPopular extends React.Component {
+    next = () => {
+        this.slider.slickNext();
+    }
+
+    previous = () => {
+        this.slider.slickPrev();
+    }
+
+    render() {
+        const {
```

Comment:
Please try to follow destructing of props:
const { productsList } = this.props;

Reply:
Done.